napari/_vispy/layers/vectors.py

```diff
@@ -4,6 +4,7 @@
 
 from ..visuals.vectors import VectorsVisual
 from .base import VispyBaseLayer
+from napari.layers.utils.layer_utils import segment_normal
```

Comment:
can you try putting this as a relative import not an absolute import? maybe that will trigger import error. we also prefer to use relative imports in these places